server/services/Mailer.js

```diff
@@ -3,24 +3,22 @@ const helper = sendgrid.mail;
 const keys = require('../config/keys');
 
 class Mailer extends helper.Mail {
-  constructor({ subject, recipients }, content) {
+  constructor(survey, content) {
     super();
 
     this.sgApi = sendgrid(keys.sendGridKey);
     this.from_email = new helper.Email('no-reply@emaily.com');
-    this.subject = subject;
+    this.subject = survey.subject;
     this.body = new helper.Content('text/html', content);
-    this.recipients = this.formatAddresses(recipients);
+    this.recipients = this.formatAddresses(survey.recipients);
```

Comment:
Since we didn't deconstruct the survey when it was passed in, we have to access the properties of the survey, so I add the `survey.`. This to me is just much easier to understand than before.